lib/addons/opa-gatekeeper/index.ts

```diff
@@ -0,0 +1,59 @@
+import { ClusterAddOn, ClusterInfo } from "../../spi";
+
+/**
+ * Configuration options for the add-on.
+ */
+export interface OpaGatekeeperAddOnProps {
+
+    /**
+     * Namespace where OPA Gatekeeper will be installed
+     * @default kube-system
+     */
+    namespace?: string;
+
+    /**
+     * Helm chart version to use to install.
+     * @default 3.6.0-beta.3
+     */
+    version?: string;
+
+    /**
+     * Values for the Helm chart.
+     */
+     values?: {
+        [key: string]: any;
+    };
+}
+
+
+/**
+ * Defaults options for the add-on
+ */
+const defaultProps: OpaGatekeeperAddOnProps = {
+    namespace: 'kube-system',
+    version: '3.6.0-beta.3',
+    values: {}
+};
+
+export class OpaGatekeeperAddOn implements OpaGatekeeperAddOn {
+
+    readonly options: OpaGatekeeperAddOnProps;
+
+    constructor(props?: OpaGatekeeperAddOnProps) {
+        this.options = { ...defaultProps, ...props };
+    }
+
+    deploy(clusterInfo: ClusterInfo): void {
+
+        const props = this.options;
+
+        clusterInfo.cluster.addHelmChart("opagatekeeper-addon", {
+            chart: "gatekeeper",
+            release: "gatekeeper",
+            repository: "https://open-policy-agent.github.io/gatekeeper/charts",
+            version: props.version,
+            namespace: props.namespace,
+            values: {}
```

Comment:
You are not leveraging values that customers can pass to your addon. You can test by passing any values (e.g. label related) to the addon, they won't be applied since you set values to `{}`

Reply:
addressed